tools/gn

```diff
@@ -1253,15 +1246,15 @@ def parse_args(args):
 
   parser.add_argument(
       '--enable-impeller-vulkan-playgrounds',
-      default=False,
+      default=True,
       action='store_true',
       help='Enable the Impeller Vulkan Playgrounds. ' +
       'The Impeller Vulkan backend needs to be enabled.'
   )
 
   parser.add_argument(
       '--enable-impeller-opengles',
-      default=False,
+      default=True,
```

Comment:
Should the default value for `--enable-impeller-vulkan` also be switched?

Reply:
Lets try it